components/QuestionMark/QuestionMark.tsx

```diff
@@ -3,17 +3,34 @@ import styled from 'styled-components';
 import { useState } from 'react';
 
 import { Tooltip } from 'components/common';
+import theme from 'styles/theme';
```

Comment:
The theme is going to be addressed by @0xcdb. Not sure if he is using this library.  Could you check it with him?